cmake/Modules/FindSYCL.cmake

```diff
@@ -27,7 +27,7 @@
 include_guard()
 
 # Try to find a DPC++ release
-# (reqrs source /opt/intel/oneapi/compilers/2023.2.0/env/vars.sh)
+# (reqrs source /opt/intel/oneapi/compilers/2024.0/env/vars.sh)
```

Comment:
same note about the oneapi dir here

Reply:
Replaced it to `/path/to/intel/oneapi/compiler`